server/planning/content_profiles/profiles/event.py

```diff
@@ -31,7 +31,7 @@ class EventSchema(BaseSchema):
     language = LanguageField()
     links = schema.ListField()
     location = schema.StringField()
-    name = StringField(required=True)
+    name = TextField(required=True, field_type="multi_line")
```

Comment:
`field_type="single_line"` might be a better default here, as names would most often than not be single line.

It might also be good to do this same change for `PlanningSchema.name` field as well

Reply:
sounds good Thanks !!